support_matrix/_topics/automation_providers.md

```diff
@@ -0,0 +1,13 @@
+### Automation  Providers
+
+The following table outlines the status of support for {{ site.data.product.title_short }} features on automation providers.
+
+#### Automation Provider Feature Support
+| Feature                        | Ansible Tower |
```

Comment:
@Fryguy should we include Embedded Ansible here?

Reply:
don't think so - as it is not really an Automation Provider or is it?

Reply:
Yeah it is actually, seem to remember @Fryguy wanting to treat it more like a general automation manager rather than something "special"

Reply:
Added Embedded Ansible and Automation Engine